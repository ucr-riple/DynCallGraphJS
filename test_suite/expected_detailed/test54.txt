{
    'global (test54.js:1:1:9:11)': [
        'anon (test54.js:1:2:9:8)'
    ],
    'anon (test54.js:8:9:8:15)': [
        'main (test54.js:2:9:7:10)'
    ],
    'main (test54.js:6:11:6:30)': [
        'anon (test54.js:5:19:5:50)'
    ],
    'anon (test54.js:5:42:5:47)': [
        'anon (test54.js:6:13:6:29)'
    ],
    'anon (test54.js:6:24:6:28)': [
        'anon (test54.js:4:19:4:49)'
    ],
    'anon (test54.js:4:42:4:47)': [
        'anon (test54.js:3:19:3:43)'
    ]
}