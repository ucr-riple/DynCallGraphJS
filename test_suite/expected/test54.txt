{ '(test54.js:1:1:9:11)':
   [ '(test54.js:1:2:9:8)' ],
  '(test54.js:8:9:8:15)':
   [ '(test54.js:2:9:7:10)' ],
  '(test54.js:6:11:6:30)':
   [ '(test54.js:5:19:5:50)' ],
  '(test54.js:5:42:5:47)':
   [ '(test54.js:6:13:6:29)' ],
  '(test54.js:6:24:6:28)':
   [ '(test54.js:4:19:4:49)' ],
  '(test54.js:4:42:4:47)':
   [ '(test54.js:3:19:3:43)' ] }